src/apps/dashboard/AppLayout.tsx

```diff
@@ -1,15 +1,15 @@
 import AppBar from '@mui/material/AppBar';
 import Box from '@mui/material/Box';
-import { useTheme } from '@mui/material/styles';
-import React, { FC, useCallback, useEffect, useState } from 'react';
+import { type Theme } from '@mui/material/styles';
```

Comment:
Hi, just getting started working with jellyfin and I don't work with React (and hence nor mui), so apologies if I'm missing something here. I haven't ever seen `import type` actually used and the docs make it sound unnecessary for most users:

> This feature is something most users may never have to think about

and 

>  In conjunction with import type, TypeScript 3.8 also adds a new compiler flag to control what happens with imports that won’t be utilized at runtime: [importsNotUsedAsValues](https://www.typescriptlang.org/tsconfig#importsNotUsedAsValues). This flag takes 3 different values:
> 
>     remove: this is today’s behavior of dropping these imports. It’s going to continue to be the default, and is a non-breaking change.

Are the `type`s here needed?

Reply:
fwiw I wanted to see if it made a difference, so I did production builds of your branch with and without the `import types`. I checksum'd each and they are the same (I'm not sure if this is true for dev builds).

Reply:
Using `input type` helps with transpilers (like esbuild/babel) as they can entirely ignore the import statement (and skip reading the file) because they don't do type checking. In large projects this can reduce build times significantly.

Reply:
Thank you for sharing, that's good to know